### ISSUE TITLE
error in postgress utils file after upgrading to deno 1.6.0

### ISSUE THREAD

**Issue by NeroStarx** [denodrivers/postgres#188]
I updated my deno version to 1.6.0, then when i tryed to run my server.ts this error message appeared and it wasn't outputing that error in deno version 1.5.4, so please any idea?

```
TS2345 [ERROR]: Argument of type 'T | undefined' is not assignable to parameter of type 'T | PromiseLike<T>'.
  Type 'undefined' is not assignable to type 'T | PromiseLike<T>'.
      resolve(value);
              ~~~~~
    at https://raw.githubusercontent.com/deno-postgres/deno-postgres/master/utils.ts:94:15

TS2345 [ERROR]: Argument of type 'T | undefined' is not assignable to parameter of type 'T | PromiseLike<T>'.
  Type 'undefined' is not assignable to type 'T | PromiseLike<T>'.
      resolve(value);
              ~~~~~
    at https://deno.land/x/postgres@v0.4.5/utils.ts:94:15
```

**Comment by saydus** [comment]
Same issue here!

**Comment by Soremwar** [comment]
This was resolved back in #187
